model/project_ref.go

```diff
@@ -1968,6 +1969,16 @@ func FindNonHiddenProjects(ctx context.Context, key string, limit int, sortDir i
 		filter[ProjectRefIdKey] = bson.M{"$gte": key}
 	}
 
+	// Add owner filter if provided
```

Comment:
nit: the comments on this + L1977 pretty much just describe what the code is doing in words, so I would recommend removing since it's redundant.

Reply:
Agreed haha, vibe coding 🤦‍♀️  i removed this in the past but i haven't figured out how to keep my agent from re-overwriting my changes later...